src/main/java/com/bootcamp/demo/service/IBookingService.java

```diff
@@ -0,0 +1,21 @@
+package com.bootcamp.demo.service;
+
+import com.bootcamp.demo.model.Booking;
+import com.bootcamp.demo.model.PaymentStatus;
+
+import java.util.Set;
+import java.util.concurrent.ExecutionException;
+
+public interface IBookingService {

```

Comment:
Is this left from the version from a branch? As in master, it was not kept due to the reasons outlined here: https://github.com/nymphadora-tonks-team/booking-jb/pull/29#discussion_r763322528

Reply:
As a follow-up since I think most conflicts were resolved. Shouldn't this be removed?